api/opencl_runtime_layer.asciidoc

```diff
@@ -13865,8 +13865,8 @@ include::{generated}/api/version-notes/clEnqueueWaitSemaphoresKHR.asciidoc[]
   * _sema_payload_list_ points to the list of values of type
     {cl_semaphore_payload_khr_TYPE} containing valid semaphore payload
     values to wait on.
-    This can be set to `NULL` or will be ignored when all semaphores in the
-    list of _sema_objects_ are of type {CL_SEMAPHORE_TYPE_BINARY_KHR}.
+    This can be set to `NULL` or will be ignored if no semaphores in the
+    list of _sema_objects_ require a payload.
```

Comment:
Nitpick: Is the **pointer** ignored if no semaphores require a payload, or is the **value that the pointer is pointing to** ignored?  In the former case the pointer can be completely bogus, since it will not be dereferenced, but in the latter case it will still need to point to valid memory.

Note: I think we have a testing gap regardless how this question is answered, since I don't see anywhere in the CTS where we currently pass a non-NULL value for the semaphore payload list, for either `clEnqueueWaitSemaphoresKHR` or `clEnqueueSignalSemaphoresKHR`.